src/main/java/seedu/duke/task/ItemList.java

```diff
@@ -159,15 +188,24 @@ public void clearTask() {
      *
      * @param index the index of the item in the list
      */
-    public void markAsDone(int index) {
+    public void markTaskAsDone(int index) {
         if (index > items.size() || index < 1) {
             Ui.dukePrint(Messages.WARNING_NO_TASK);
         } else {
-            items.get(index - 1).markAsDone();
+            items.get(index - 1).markAsDoneOrReturn();
             Ui.dukePrint(Messages.MESSAGE_DONE + items.get(index - 1).getDescription());
         }
     }
 
+    public void markBookAsReturned(int index) {
+        if (index > items.size() || index < 1) {
+            Ui.dukePrint(Messages.WARNING_NO_TASK);
+        } else {
+            items.get(index - 1).markAsDoneOrReturn();
+            Ui.dukePrint(Messages.MESSAGE_RETURNED + items.get(index - 1).getDescription());
+        }
+    }
+
```

Comment:
Same

Reply:
Alright, will make the adjustment for all

Reply:
Yup I think it LGTM except that Book related functionality should be moved from `Item` to `Book` and same for `BookList` related functionality from `Item`.